### PR TITLE
feat: freeze jina version in hub build

### PR DESCRIPTION
**Pull request by JoanFM**
**Changes introduced**
Freeze jina version to the core version that builds the hub image

### CODE REVIEW

**Review by nan-wang — APPROVED** [review]
LGTM👍 Maybe we can add a unit test?

**Comment by JoanFM** [comment]
@nan-wang does not this conflict with ur PR in action builder?

File: jina/docker/hubio.py [hunk 1]
```diff
@@ -504,15 +507,37 @@ def _write_summary_to_file(self, summary: Dict) -> None:
             json.dump(summary, f)
         self.logger.debug(f'stored the summary from build to {file_path}')
 
+    def _freeze_jina_version(self) -> None:
+        import pkg_resources
+        from jina import __version__
+        jina_core_version = __version__
```
Comment (nan-wang) [inline jina/docker/hubio.py:513]: duplicated with Line 16: `from .. import __version__ as jina_version`
```suggestion
```

**Review by nan-wang — CHANGES_REQUESTED** [review]

**Comment by nan-wang** [comment]
> @nan-wang does not this conflict with ur PR in action builder?

Nope. This is consistent with current behaviors. The hub-builder calls `hubio` for building the image and therefore the `jina_version` depends on which base image is used. 

However, your concern is valid. The Jina versions used in hub-updater and hub-builder are implicitly coupled.

**Review by nan-wang — APPROVED** [review]
LGTM👍

**Pull request merged**